Description: switch test from ava to tape
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2020-02-19

--- a/test.js
+++ b/test.js
@@ -1,5 +1,5 @@
-import test from 'ava';
-import arrify from '.';
+const test = require('tape');
+const arrify = require('.');
 
 test('main', t => {
 	t.deepEqual(arrify('foo'), ['foo']);
@@ -10,4 +10,5 @@
 
 	const fooArray = ['foo'];
 	t.is(arrify(fooArray), fooArray);
+	t.end();
 });
